core/jobs_test.py

```diff
@@ -949,6 +1176,106 @@ def test_events_coming_in_while_batch_job_is_running(self):
             with self.assertRaises(base_models.BaseModel.EntityNotFoundError):
                 stats_models.ExplorationAnnotationsModel.get(self.EXP_ID)
 
+    def test_cannot_start_new_job_while_existing_job_still_running(self):
+        with self.swap(
+            jobs_registry, 'ALL_CONTINUOUS_COMPUTATION_MANAGERS',
+            self.ALL_CC_MANAGERS_FOR_TESTS
+            ):
+            StartExplorationEventCounter.start_computation()
+            with self.assertRaisesRegexp(
+                Exception,
+                'Attempted to start computation StartExplorationEventCounter, '
+                'which is already running'):
+                StartExplorationEventCounter.start_computation()
+
+            self.process_and_flush_pending_tasks()
+            StartExplorationEventCounter.stop_computation('admin_user_id')
+
+    def test_get_continuous_computations_info_with_existing_model(self):
+        job_models.ContinuousComputationModel(
+            id='StartExplorationEventCounter').put()
+        continuous_computations_data = jobs.get_continuous_computations_info(
+            [StartExplorationEventCounter])
+
+        self.assertEqual(len(continuous_computations_data), 1)
```

Comment:
Any reason why you can't replace all these lines with

    self.assertEqual(continuous_computations_data, ...) ?

Reply:
used assertDictContainsSubset as some fields cannot be asserted